test/lib/environment.spec.js

```diff
@@ -17,6 +17,13 @@ describe('environment', () => {
     });
   });
 
+  describe('apiUrl initialize empty', ()=>{
+    it('undefined apiUrl remains undefined', ()=>{
```

Comment:
Upon further checks, the value that comes when the instance is not required is Boolean `false`, not `undefined`, so I think this test should reflect this.